### PR TITLE
glossary stub

### PR DESCRIPTION
**Pull request by psteinb**
this PR adds a cross off list to the glossary based in #37

### CODE REVIEW

File: reference.md [hunk 1]
```diff
@@ -5,4 +5,26 @@ permalink: /reference/
...
+- [ ] [High availability computing](https://en.wikipedia.org/wiki/High_availability)
+- [ ] [High performance computing](https://en.wikipedia.org/w/index.php?title=High-performance_computing&redirect=no)
+- [ ] [Interconnect](https://en.wikipedia.org/wiki/Supercomputer_architecture)
+- [ ] [Node](https://en.wikipedia.org/wiki/Node_(computer_science))
+- [ ] [Parallel](https://en.wikipedia.org/wiki/Parallel_computing)
+- [ ] [Serial](https://en.wikipedia.org/wiki/Serial_computer)
+- [ ] [Server](https://en.wikipedia.org/wiki/Server_(computing))
+- [ ] [Shared memory](https://en.wikipedia.org/wiki/Shared_memory)
+- [ ] [Slurm](https://en.wikipedia.org/wiki/Slurm_Workload_Manager)
+- [ ] [Supercomputer](https://en.wikipedia.org/wiki/Supercomputer) -- "[a major scientific instrument](https://www.hpcnotes.com/2015/10/essential-analogies-for-hpc-advocate.htmlhttps://www.hpcnotes.com/2015/10/essential-analogies-for-hpc-advocate.html)"
```
Comment (ocaisa) [inline reference.md]: https://www.hpcnotes.com/2015/10/essential-analogies-for-hpc-advocate.html is (now?) a broken link :(

Comment (ocaisa) [reply reference.md]: Whoops, must have copy-pasted wrong, abort comment

File: reference.md [hunk 2]
```diff
@@ -5,4 +5,26 @@ permalink: /reference/
 
 ## Glossary
 
-FIXME
+The following list captures terms that need to be added to this glossay. This is a great way to contribute.
```
Comment (tkphd) [inline reference.md]: glossary, not glossay

**Comment by psteinb** [comment]
acknowledged. Feel free to review and potentially merge.

**Comment by tkphd** [comment]
Much better. Thanks, @psteinb! And good catch @ocaisa, fixed the url in #37 as well.

**Comment by psteinb** [comment]
@tkphd @ocaisa please review again. I suggest to merge though so that we can go ahead even if the cluster definition I just added will not be the final one.

**Comment by tkphd** [comment]
Looking at the [git-novice reference.md](https://raw.githubusercontent.com/swcarpentry/git-novice/gh-pages/reference.md), they use a slick Jekyll trick to automatically create direct-link tags. Recommend reformatting (again, sorry) to match their pattern:

```markdown
...

## Glossary

{:auto_ids}
cluster
:     a collection of computers configured to enable collaboration on a common task by
      means of purposefully configured hardware (*e.g.*, networking) and software 
      (*e.g.* workload management).
supercomputer
:    "[a major scientific instrument](https://www.hpcnotes.com/2015/10/essential-analogies-for-hpc-advocate.html)"

...
```

Otherwise, this stub LGTM.

**Comment by psteinb** [comment]
Lovely, I’ll look into this on Monday.

**Pull request merged**